test-runner.js

```diff
@@ -0,0 +1,5 @@
+const testsContext = require.context('./tests', true, /\.js$/);
+const componentsContext = require.context('./src/', true, /components\/.*\.js$/);
```

Comment:
We can add anything else that we need here but if you add `src/**/*.js` the tests blow-up.